packages/bpk-component-input/src/withOpenEvents.tsx

```diff
@@ -168,14 +167,7 @@ const withOpenEvents = <P extends object>(InputComponent: ComponentType<P>) => {
 
       const eventHandlers: EventHandlers = {
         onClick: withEventHandler(onOpen, onClick),
-        onKeyDown: withEventHandler(
-          handleKeyEvent(KEYCODES.ENTER, onOpen),
-          onKeyDown,
-        ),
-        onKeyUp: withEventHandler(
-          handleKeyEvent(KEYCODES.SPACEBAR, onOpen),
-          onKeyUp,
-        ),
+        onKeyDown: withEventHandler(handleKeyEvent(onOpen), onKeyDown),
```

Comment:
Why does it complain about types? KeyboardEvent should be a subtype of UIEvent.

Reply:
I don't know why it is complaining about this, searching online isn't helpful for the typing of this and just only points at what KeyboardEvent is

Reply:
Discussed with @olliecurtis , this looks like the best solution